spring-framework/5.3.14.wso2v1/pom.xml

```diff
@@ -0,0 +1,104 @@
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.orbit.org.springframework.ws</groupId>
+    <artifactId>spring.framework</artifactId>
+    <packaging>bundle</packaging>
+    <name>spring.framework.wso2</name>
+    <version>5.3.14.wso2v1</version>
+    <description>
+        This bundle will export packages from spring framework
+    </description>
+    <url>https://wso2.org</url>
+
+   <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>https://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
+            <id>wso2.snapshots</id>
+            <name>WSO2 Snapshot Repository</name>
+            <url>https://maven.wso2.org/nexus/content/repositories/snapshots/</url>
+        </snapshotRepository>
+   </distributionManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-beans</artifactId>
+			<version>${version.spring}</version>
+			<optional>true</optional>
+		</dependency>
+		<dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+			<version>${version.spring}</version>
+			<optional>true</optional>
+        </dependency> 
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-tx</artifactId>
+			<version>${version.spring}</version>
+			<optional>true</optional>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>5.1.4</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.springframework.*; version=${exp.pkg.version.spring},
+                        </Export-Package>
+                        <Private-Package>
+                            org.springframework.*
+                        </Private-Package>
+                        <Import-Package>
+							!org.springframework.*;,
```

Comment:
Formatting looks off.

Reply:
The formatting issues were fixed by this PR -  https://github.com/wso2/orbit/pull/646